AnnoyancesFilter/Popups/sections/subscriptions_specific.txt

```diff
@@ -6732,6 +6732,8 @@ flightradar24.com##[data-id="7d6f55d"][data-element_type="widget"]
 flightradar24.com##[data-id="a2aa32b"][data-element_type="widget"]
 flightradar24.com##[data-id="a5b8151"][data-element_type="widget"]
 flightradar24.com##[data-id="35ae40a"][data-element_type="widget"]
+! subscription at the bottom
+cloudflare.com##.bg-blue4
```

Comment:
This rule will hide all elements with blue background.

I would use a rule like this, it is tied to the content of the element.
`cloudflare.com##.mb7 > div.bg-blue4:has(> section > form input#Email)`
Please add this rule under this comment:
!
! SECTION: Subscriptions - Regular rules
!

Reply:
Also if you want to use a comment, please always use GitHub issue link if available.